test/channels_api_SUITE.erl

```diff
@@ -102,3 +103,37 @@ delete_channel(Config) ->
     {ok, 204, _, _} = test_utils:api_request(delete, Token, "/api/channels/my_delete_channel"),
     {ok, 404, _, _} = test_utils:api_request(delete, Token, "/api/channels/my_delete_channel"),
     ok.
+
+create_delete_channel_notification(Config) ->
```

Comment:
you also need to test the edit part: if you add a new target to the channel, the notification should go only to the new one; if you remove one, only the removed should receive the notification.